### ISSUE TITLE
Ignoring invalid email address: sendmail

### ISSUE THREAD

**Issue by dregad** [axllent/mailpit#152]
My mailpit.log file is filled with messages like this

```
WARN[2023/08/08 02:24:57] [smtpd] ignoring invalid email address: sendmail
```

Mail is sent by a PHP application using PHPMailer, configured to use PHP mail(), with *php.ini* setup per https://github.com/axllent/mailpit/wiki/Configuring-sendmail, i.e.
`sendmail_path = /opt/homebrew/bin/mailpit sendmail`

Mail header example
```
Received: from localhost (localhost [::1])
        by damien.local (Mailpit) with SMTP
        for <sendmail>; Tue,  8 Aug 2023 02:24:57 +0200 (CEST)
To: NfaldtBtNL@somedomain.com
Subject: =?us-ascii?Q?[Mantis_DEV_2.26.0-dev_(php72-8394cc5)]_Account?=
 =?us-ascii?Q?_registration?=
Date: Tue, 8 Aug 2023 00:24:57 +0000
From: "Mantis DEV 2.26.0-dev (php72-8394cc5)" <mantis@localhost.com>
Message-ID: <1OE6JgDTg7nGmHerVfsq7WAnv5G4RpspHoogXk2aE@localhost>
X-Mailer: PHPMailer 6.8.0 (https://github.com/PHPMailer/PHPMailer)
Auto-Submitted: auto-generated
X-Auto-Response-Suppress: All
MIME-Version: 1.0
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable
```

Mailpit 1.8.0

**Comment by axllent** [comment]
This should have been fixed in v1.8.1. Could you please upgrade your Mailpit and test again?

**Comment by dregad** [comment]
My apologies, I should have checked for a new release and tested before reporting an issue here.

I confirm that the problem is fixed in 1.8.1 indeed. 

Sorry about the noise and many thanks for your help and for making Mailpit !

**Comment by axllent** [comment]
Not a problem at all, and I'm glad you like Mailpit 👍
